lib/hound/helpers.ex

```diff
@@ -24,6 +24,8 @@ defmodule Hound.Helpers do
     quote do
       setup do
         Hound.start_session
+        on_exit fn-> Hound.end_session end
```

Comment:
`on_exit` does not run in the same in the same process as the test, so I think we need to explicitly pass
the pid:

```elixir
pid = self()
on_exit fn -> Hound.end_session(pid) end
```

http://elixir-lang.org/docs/stable/ex_unit/ExUnit.Callbacks.html